tests/tests/test-mock-hrmp.ts

```diff
@@ -237,13 +237,13 @@ describeDevMoonbeam("Mock XCM - receive horizontal transfer", (context) => {
     await context.createBlock();
 
     // Make sure the state has ALITH's foreign parachain tokens
-    let alith_dot_balance = (
-      (await context.polkadotApi.query.assets.account(assetId, alith.address)) as any
-    )
-      .unwrap()
-      ["balance"].toBigInt();
+    let alith_dot_balance = (await context.polkadotApi.query.assets.account(
+      assetId,
+      alith.address
+    )) as any;
 
-    expect(alith_dot_balance).to.eq(10n * FOREIGN_TOKEN);
+    // The message execution failed
+    expect(alith_dot_balance.isNone).to.eq(true);
```

Comment:
alternate [syntax](https://www.chaijs.com/api/bdd/#method_true):
```suggestion
    expect(alith_dot_balance.isNone).to.be.true;
```